.eslintrc

```diff
@@ -39,6 +39,7 @@
     "@typescript-eslint/no-empty-function": "off",
     "@typescript-eslint/no-explicit-any": "off",
     "@typescript-eslint/no-non-null-assertion": "off",
+    "@typescript-eslint/no-inferrable-types": "error",
```

Comment:
I'm guessing that adding this caused the current Lint error

![image](https://github.com/ghiscoding/slickgrid-universal/assets/643976/6f928f3c-9478-4b57-8749-a3c731ffcb6a)

Reply:
ups sry forgot to check before pushing. do you mind adding husky with a pre-push hook running the linter to circumvent these kind of issues?

Reply:
I've never tried Husky, I never took the time to set it up but feel free to add it, it would certainly help in these situations

Reply:
I've added it now with a pre-push hook. You should run pnpm install when checking it out. Next time you try to push and there are still linter errors, it will abort the push.